.env.beta

```diff
@@ -0,0 +1,4 @@
+REACT_APP_GA_ID=UA-164162157-1
+REACT_APP_PANTRY_FINDER_API=https://pantry-finder-api.beta.freshtrak.com/
+
+GENERATE_SOURCEMAP=false
```

Comment:
@ashikvarma11 added the sourcemaps env var here so that there's a  central place for environment specific build configuration. Thanks for the tip.